contracts/OpenSTValue.sol

```diff
@@ -222,6 +223,8 @@ contract OpenSTValue is OpsManaged, Hasher {
             unlockHeight: unlockHeight,
             hashLock:     _hashLock
         });
+        /** Add intent hash to intents mapping */
```

Comment:
correct comment type, and insert a whiteline above

this is an important line in the code, so it deserve a more elaborate comment, eg:
"Store the staking intent hash directly in storage of OpenSTValue so that a Merkle proof can be generated for active staking intents"

Reply:
Done!